app/javascript/controllers/form_validation_controller.js

```diff
@@ -61,6 +63,26 @@ export default class FormValidation extends ValidationController {
       'focus:border-red-500',
     );
   }
+
+  static removeError(el) {
+    const errorField = el.parentElement.nextElementSibling;
+    errorField.classList.add('hidden');
+    errorField.textContent = '';
+    el.classList.remove(...el.classList);
+    el.classList.add(
+      'block',
```

Comment:
Having the default form styles duplicated here introduces a little bit of brittleness, we'd need to remember to update here and the builder if we ever need to update the form field designs. 

We could alternatively take the invalid field classes on [line 59](https://github.com/TheOdinProject/theodinproject/blob/54d17a8b714274c64951ceb2ababf45f7890ed2d/app/javascript/controllers/form_validation_controller.js#L59) and put them into a const above the class definition:

```javascript
const invalidFieldClasses = [
  'border-red-300',
  'text-red-900',
  'placeholder-red-300',
  'focus:ring-red-500',
  'focus:border-red-500',
]

export default class FormValidation extends ValidationController {
  ...
}
```

Then we can simplify adding and removing error styles
```javascript
static applyErrorStylesTo(el) {
    el.classList.add(...invalidFieldClasses);
  }

  static removeError(el) {
    const errorField = el.parentElement.nextElementSibling;
    errorField.classList.add('hidden');
    errorField.textContent = '';
    el.classList.remove(...invalidFieldClasses);
  }

```

Reply:
Fixed with [13a3cd1](https://github.com/TheOdinProject/theodinproject/pull/3380/commits/13a3cd1e5295b1506531b7cf5635014c7e0c5561)!